transports/upgrades/asq-8to9.md

```diff
@@ -1,16 +1,57 @@
 ---
 title: Azure Storage Queues Transport Upgrade Version 8 to 9
 summary: Migration instructions on how to upgrade Azure Storage Queues Transport from Version 8 to 9.
-reviewed: 2029-11-09
+reviewed: 2020-11-12
 component: ASQ
 related:
 - transports/azure-storage-queues
 - nservicebus/upgrades/7to8
 isUpgradeGuide: true
 upgradeGuideCoreVersions:
- - 8
+ - 7
 ---
 
+## Package name
+
+This package was formerly known as `NServiceBus.Azure.Transports.WindowsAzureStorageQueues`.
+As of version 9 the package name is `NServiceBus.Transport.AzureStorageQueues`.
+
+## Types from namespace `NServiceBus.Azure.Transports.WindowsAzureStorageQueues` moved to `NServiceBus.Transport.AzureStorageQueues`
+
+Certain advanced configuration APIs have been moved from the namespace `NServiceBus.Azure.Transports.WindowsAzureStorageQueues` to `NServiceBus.Transport.AzureStorageQueues`Code must to be adjusted accordingly. A straight forward way is to search and replace
+
+```
+using NServiceBus.Azure.Transports.WindowsAzureStorageQueues;
+```
+
+with
+
+```
+using NServiceBus.Transport.AzureStorageQueues;
+```
+
+## Move to .NET 4.7.2
+
+The minimum .NET Framework version for version 4 is [.NET Framework 4.7.2](https://dotnet.microsoft.com/download/dotnet-framework/net472).
```

Comment:
> The minimum .NET Framework version for version 4 is .NET Framework 4.7.2

does not read well. It reads as "the minimum .NET FW version for version 4 _of the transport_. How about

```suggestion
The minimum .NET Framework version is [.NET Framework 4.7.2](https://dotnet.microsoft.com/download/dotnet-framework/net472).
```